src/oemof/tabular/facades.py

```diff
@@ -29,11 +32,72 @@
 from oemof.solph.plumbing import sequence
 from oemof.tools.debugging import SuspiciousUsageWarning
 
-
 # Switch off SuspiciousUsageWarning
 warnings.filterwarnings("ignore", category=SuspiciousUsageWarning)
 
 
+def kwargs_to_parent(cls):
+    r"""
+    Decorates the __init__ of a given class by first
+    passing args and kwargs to the __init__ of the parent
+    class.
+
+    Parameters
+    ----------
+    cls : Class with an __init__ to decorate
+
+    Returns
+    -------
+    cls : Class with decorated __init__
+    """
+    original_init = cls.__init__
+
+    def new_init(self, *args, **kwargs):
+
+        super(cls, self).__init__(*args, **kwargs)
```

Comment:
Looking at the problem with `conversion_factor_full_condensation`, I found out: 
*args and **kwargs do not contain the default values yet. Therefore, if an argument is not provided, the default is not passed to the parent class, as it is set in the original_init.

Reply:
Addressed in [b51e94e](https://github.com/oemof/oemof-tabular/pull/44/commits/b51e94e4ce996be82c57d9eff0d8a7b3774dabac)